package/etc/conf.d/filters/cisco/esa.conf

```diff
@@ -0,0 +1,44 @@
+filter f_cisco_esa {
+    match("cisco_esa_12_5", value("fields.sc4s_vendor_product") type(glob))
```

Comment:
if we can avoid it we don't want this to be product-version specific because admins upgrade stuff and don't tell the log admins

Reply:
Here we want to differentiate the events based on the host and so we have assigned the sc4s_vendor_product as cisco_esa_12_5. Hence all the logs coming from that host will be assigned to this vendor_product. Do you suggest we change that? and If yes, What is the condition that we put?